code/frontend/MakeItSo/Shared/Features/ReminderList/ViewModels/ReminderDetailsViewModel.swift

```diff
@@ -129,12 +134,11 @@ class ReminderDetailsViewModel: ObservableObject {
     set {
       if newValue == true {
         reminder.dueDate = Date()
-        isShowingDatePicker = true
-      }
-      else {
-        hasDueTime = false
+        setPickerState(.date)
+      } else {
```

Comment:
Personally, I am a fan of "egyptian style":

if condition {
  // do stuff
}
else { // <-- else is on new line, and (most importantly) parallel with the initial `if`
  // do other stuff
}

Reply:
Stylistically, I like that too, but generally I look to try to shorten my code as long as it remains clearly understood.
I am a fan of the old school Nassi Schneiderman style of coding, which I have adapted to mean that generally a function/view/discrete piece of code should all fit on the screen. If it doesn't, it needs refactoring.

Reply:
Appreciate your stylist conventions should take precedence in your repo!